modules/highgui/src/window.cpp

```diff
@@ -165,7 +165,7 @@ CV_IMPL double cvGetWindowProperty(const char* name, int prop_id)
     }
 }
 
-cv::Rect cvGetWindowImageRect(const char* name)
+CvRect cvGetWindowImageRect(const char* name)
 {
     if (!name)
         return cv::Rect(-1, -1, -1, -1);
```

Comment:
You will need a CvRect here too ;)

Reply:
Where? In the actual function? I think it works ok without changing the cv::Rect on line 171, it compiles fine. I imagine cv::Rect and CvRect are the same under the hood, one just isn't usable in C. But Let me know if I'm mistaken!